File: ProyectoFinal/ProyectoFinal/Parametros/Salida/ListarAlertasOut.cs
using ProyectoFinal._2_Dominio.Entidades;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal.Parametros.Salida
{
    public class ListarAlertasOut : ParametroBaseOut
    {
        public List<Alerta> Alertas { get; set; }
    }
}

File: Interfaz/ProyectoFinal/src/app/Parametros/Salida/ListarAlertasOut.ts
import { Alerta } from "src/app/Components/Alerta/alerta";
import { ParametroBaseOut } from "./ParametroBaseOut";

export class ListarAlertasOut extends ParametroBaseOut{
    
    private alertas: Alerta[];

    constructor()
    {
        super();
        this.alertas = [];
    }

    public get Alertas(): Alerta[] {
        return this.alertas;
    }
    public set Alertas(value: Alerta[]) {
        this.alertas = value;
    }
}